Gruntfile.js

```diff
@@ -314,6 +314,25 @@ module.exports = function(grunt) {
         files: {
           'test/actual/pages_metadata/': ['test/fixtures/pages/blog/index.hbs']
         }
+      },
+      // Using glob pattern for layouts
+      glob_layout: {
+        options: {
+          layouts: 'test/fixtures/layouts/**/*.hbs',
+          layoutdir: undefined
```

Comment:
Okay, sure that's fine. Let's say if `layoutdir`, it will serve as `cwd` for the glob. If it's undefined the current directory will serve as `cwd`.